src/main/kotlin/org/rust/ide/spellchecker/StringLiteralTokenizer.kt

```diff
@@ -22,9 +22,8 @@ class StringLiteralTokenizer : EscapeSequenceTokenizer<LeafPsiElement>() {
         fun processTextWithEscapeSequences(element: LeafPsiElement, text: String, consumer: TokenConsumer) {
             val unescapedText = StringBuilder()
             val offsets = IntArray(text.length + 1)
-            CodeInsightUtilCore.parseStringCharacters(text, unescapedText, offsets)
-
-            processTextWithOffsets(element, consumer, unescapedText, offsets, 1)
```

Comment:
@Turbo87 your screenshots work correctly, because this code isn't evaluated in them. IntelliJ checks if there are any `\` in the token, and fires `processTextWithEscapeSequences` only when some are found.

I found this bug when I wrote `string_literals_with_escapes.rs`